amundsen_application/static/js/components/common/FlashMessage/styles.scss

```diff
@@ -0,0 +1,22 @@
+@import 'variables';
+
+.flash-message {
+  $spacer-size: 8px;
```

Comment:
I can make this a global variable -- for me it encourages the use of multipliers of `$spacer-size` moving forward to hold myself accountable to the designer's spacing.

I also think having the dedicated spacing variables adds to the concept of trying to establish some support/guidelines that contributors can follow to add new components for without us needing to provide them a solid design. Essentially if someone tries to use as many established variables as possible in terms of colors, fonts, and now spacing -- they will most likely be able to create something that at least matches our application's look and feel without needing dedicated design support from us.

Reply:
Made it a global

Reply:
Tactically I wonder if it would be better to define additional spacing values. For example we would currently need to do:

```
margin: ($spacer-size * 2) ($spacer-size * 3);
```

vs
```
$spacer-1: $spacer-size;
$spacer-2: $spacer-size * 2;
$spacer-3: $spacer-size * 3;
...

margin: $spacer-2 $spacer-3;
```

Reply:
I'll go ahead and add `$spacer-2` and `$spacer-3`. I'll also start using them in new work when we merge down all the things.